assets/media-gallery.js

```diff
@@ -14,7 +14,7 @@ if (!customElements.get('media-gallery')) {
       this.elements.thumbnails.querySelectorAll('[data-target]').forEach((mediaToSwitch) => {
         mediaToSwitch.querySelector('button').addEventListener('click', this.setActiveMedia.bind(this, mediaToSwitch.dataset.target, false));
       });
-      if (this.dataset.desktopLayout !== 'stacked' && this.mql.matches) this.removeListSemantic();
+      if (this.dataset.desktopLayout !== 'stacked' || this.dataset.desktopLayout !== 'columns' && this.mql.matches) this.removeListSemantic();
```

Comment:
I do like that better as it does improve readability. I can add in this in 👍🏼 

Since we're essentially checking a string it got me curious, would there be any downsides to using the [includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) method here? I it works and confirmed that both the "Thumbnails" and "Thumbnails slider" desktop layout setting options produce `thumbnail` and `thumbnail_slider` respectively: 

```javascript
if (this.dataset.desktopLayout.includes('thumbnail') && this.mql.matches) this.removeListSemantic();
```